apps/emqx/i18n/emqx_schema_i18n.conf

```diff
@@ -1901,6 +1901,347 @@ base_listener_acceptors {
     }
 }
 
+fields_mqtt_quic_listener_max_bytes_per_key {
+    desc {
+        en: "Maximum number of bytes to encrypt with a single 1-RTT encryption keybefore initiating key update. Default: 274877906944"
```

Comment:
```suggestion
        en: "Maximum number of bytes to encrypt with a single 1-RTT encryption key before initiating key update. Default: 274877906944"
```

Reply:
thanks, I auto generated them from last commit the newlines are forgotten. :smile: